utils/command_build_clib.py

```diff
@@ -116,21 +117,37 @@ def build_libraries(self, libraries):
             """
             newer_than_lib = newer_group(objects, c_library_filename, missing="newer")
             if force_build or newer_than_lib:
-                self.compiler.link_shared_lib(objects,
-                                              lib_name,
-                                              output_dir=self.build_clib,
-                                              libraries=libraries,
-                                              library_dirs=library_dirs,
-                                              runtime_library_dirs=runtime_library_dirs,
-                                              extra_preargs=extra_preargs + extra_link_preargs,
-                                              extra_postargs=extra_link_postargs,
-                                              debug=self.debug,
-                                              build_temp=self.build_temp,
-                                              target_lang=language)
+                # TODO very brute way, need to refactor
```

Comment:
it looks like `setuptools` is useless on Windows.
Perhaps it is better to move to cmake (at least for C++ backend). I mean to create CMakeFile in the `backend` directory and call it from `setup.py`